<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCHEConfig</class>
 <widget class="QDialog" name="PCHEConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>718</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>105</y>
     <width>431</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/PCHEImg1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/PCHEImg2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>83</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>P Outlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pAirOut"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>75</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>T Inlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tAirIn"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>160</y>
     <width>83</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>T Outlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tAirOut"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Number of ZigZags</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="nbZigZag"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>275</y>
     <width>81</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="lBox">
      <property name="suffix">
       <string> mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>402</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>T Inlet Channel 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tWIn1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>504</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>P Inlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pWIn"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>608</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>T Inlet Channel 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tWIn2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>399</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>T Outlet Channel 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tWOut1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>533</x>
     <y>511</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>P Outlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pWOut"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_13">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>438</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Channel Height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="chHeight">
      <property name="suffix">
       <string> mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_14">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>438</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Channel Width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="chWidth">
      <property name="suffix">
       <string> mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_15">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>618</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_17">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Channel number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="nbChannels"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>82</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-- Air --</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>348</y>
     <width>118</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-- Water --</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_16">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_18">
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Flow Rate Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="flowRate"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_17">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>264</y>
     <width>91</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_19">
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>T ambient Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tAmb"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_18">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>388</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_20">
    <item>
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Angle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="angle">
      <property name="suffix">
       <string> °</string>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_19">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>388</y>
     <width>84</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_21">
    <item>
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Wall thickness</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="wThick">
      <property name="suffix">
       <string> mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>75</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>P Inlet Channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pAirIn"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_20">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>100</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Error (P Inlet)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="pAirInError">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> bar</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>207</y>
     <width>96</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_22">
    <item>
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Error (T Inlet)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="tAirInError">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> °C</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_22">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>305</y>
     <width>99</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_24">
    <item>
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Error (Flow Rate)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="flowRateError">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> g/s</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_23">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>96</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_25">
    <item>
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>Error (T ambient)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="tAmbError">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> °C</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_24">
   <property name="geometry">
    <rect>
     <x>577</x>
     <y>90</y>
     <width>100</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_26">
    <item>
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>Error (P Outlet)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="pAirOutError">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> bar</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_25">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>206</y>
     <width>96</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_27">
    <item>
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>Error (T Outlet)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="tAirOutError">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> °C</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_27">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>448</y>
     <width>102</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_29">
    <item>
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>Error (T Inlet 1)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="tWIn1Error">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> °C</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_28">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>100</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_30">
    <item>
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Error (P Inlet)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="pWInError">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> bar</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_29">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>652</y>
     <width>96</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_31">
    <item>
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>Error (T Inlet 2)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="tWIn2Error">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> °C</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_30">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>445</y>
     <width>96</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_32">
    <item>
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>Error (T Outlet 1)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="tWOut1Error">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> °C</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_31">
   <property name="geometry">
    <rect>
     <x>533</x>
     <y>557</y>
     <width>100</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_33">
    <item>
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>Error (P Outlet)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="pWOutError">
      <property name="prefix">
       <string>±</string>
      </property>
      <property name="suffix">
       <string> bar</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>102</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Diff. P Channel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="diffP"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_28">
      <item>
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Error (Diff. P)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="diffPError">
        <property name="prefix">
         <string>±</string>
        </property>
        <property name="suffix">
         <string> bar</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>608</y>
     <width>98</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>T Outlet Channel 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="tWOut2"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_34">
      <item>
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>Error (T Outlet 2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="tWOut2Error">
        <property name="prefix">
         <string>±</string>
        </property>
        <property name="suffix">
         <string> °C</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_26">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_23">
    <item>
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Mass flow - water</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="massFlowWater">
      <property name="prefix">
       <string/>
      </property>
      <property name="suffix">
       <string> g/s</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_32">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>650</y>
     <width>92</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_35">
    <item>
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>Time Interval</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="timeInterval">
      <property name="suffix">
       <string> s</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PCHEConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PCHEConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
